LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 LogDNA, Inc.
+Copyright (c) 2018 LogDNA, Inc.
 
```

Comment:
@smusali, why the 2018 date?

Reply:
@tanberry, we try to keep the date of the creation in the license file